desktop/src/app/components/widgets/changes-history-modal.html

```diff
@@ -0,0 +1,61 @@
+<div class="modal-header">
+    <h5  class="modal-title" i18n="@@widgets.historyModal.header">
+        Bearbeitungshistorie der <b>{{document.resource.identifier}}</b>
+    </h5>
+
+</div>
+
+<div id="changes-history-modal-body" class="modal-body" >
+    <div class="resource-creation-info-row">
+        <span class="resource-creation-info-label" i18n="@@resource.creator.name">Erstellt von: </span>
+        <span class="resource-creation-info-text">{{document.created.user | slice:0:32}}</span>
+    </div>    
+    <div class="resource-creation-info-row">
+        <span class="resource-creation-info-label" i18n="@@resource.creation.date">Erstellungsdatum: </span>
+        <span class="resource-creation-info-text">{{formatDateTime(document.created.date)}}</span>
+    </div>
+    
+    
+
+    <div *ngIf="document.modified[0].user.length>=1; else noChangeTemplate">
+        <br />
```

Comment:
Instead of using br elements, it's better to adjust the scss rules to add margins, padding etc.